fr/index.html

```diff
@@ -29,7 +29,8 @@ <h1 class="post-title">Jérémy</h1>
         'pensif',
         'optimiste',
         'en attente',
-        'équanime'
+        'équanime',
+        'Hi Greg'
```

Comment:
```suggestion
        'salut Greg'
```
you'll want to put the english version over [here](https://github.com/jtcohen6/jtcohen6.github.io/blob/c828576d81f1838157289fce3fd9e34bfbf4ffd1/index.html#L23)

Reply:
Ah I was wondering why their was an fr filepath :)